adoc/admin-storage-vsphere.adoc

```diff
@@ -0,0 +1,248 @@
+= vSphere Storage
```

Comment:
The `adoc/admin-storage-vsphere.adoc` is a new page, any link to the main chapter?

Reply:
done